Jenkinsfile

```diff
@@ -15,16 +15,28 @@
  * limitations under the License.
  */
 @Library('github.com/wooga/atlas-jenkins-pipeline@1.x') _
+
 pipeline {
     environment {
         SONAR_TOKEN = credentials("atlas_plugins_sonar_token")
         SONAR_PROJECT_NAME = "atlas-jenkins-pipeline"
         SONAR_PROJECT_KEY = "wooga_atlas-jenkins-pipeline"
+        //generates ATLAS_GITHUB_INTEGRATION_USR and ATLAS_GITHUB_INTEGRATION_PSW
+        ATLAS_GITHUB_INTEGRATION = credentials('github_integration')
+        ATLAS_GITHUB_INTEGRATION_USERNAME = "${ATLAS_GITHUB_INTEGRATION_USR}"
+        ATLAS_GITHUB_INTEGRATION_PASSWORD = "${ATLAS_GITHUB_INTEGRATION_PSW}"
+        GRGIT_USER = "${ATLAS_GITHUB_INTEGRATION_USR}"
+        GRGIT_PASS = "${ATLAS_GITHUB_INTEGRATION_PSW}"
     }
+    parameters {
+        choice(name: 'releaseType', choices: ['NONE', 'MAJOR', 'MINOR', 'PATCH'],
+                description:  "Pick a release type, or NONE if this run shouldn't generate a release")
+    }
+
     agent any
     stages {
         stage("Check") {
-            steps { gradleWrapper "check" }
+            steps { gradleWrapper "check --info --stacktrace" }

```

Comment:
Check out how we do this for the other pipelines. Essentially you need to add this to the pipeline parameters:

```groovy
choice(choices: ["", "quiet", "info", "warn", "debug"], description: 'Choose the log level', name: 'LOG_LEVEL')
booleanParam(defaultValue: false, description: 'Whether to log truncated stacktraces', name: 'STACK_TRACE')
booleanParam(defaultValue: false, description: 'Whether to refresh dependencies', name: 'REFRESH_DEPENDENCIES')
```

And then the gradleWrapper step takes the values from the env.

Reply:
Done.